### ISSUE TITLE
HorizontalTabs: add arrow-key keyboard navigation

### ISSUE THREAD

**Issue by DanEastBentley** [iTwin/iTwinUI-react#177]
#### Feature
For keyboard accessibility, the user should be able to use the left & right arrow keys to activate the tabs. The tabs also need some type of focus indication so the user knows when the left/right arrow key will activate the adjacent tab.

**Comment by veekeys** [comment]
We are aware, mentioned in #173.
I guess better have it as separate issue, easier to follow.

**Comment by mayank99** [comment]
Now available in iTwinUI-react 1.12.0 🎉
